GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.Designer.cs

```diff
@@ -143,6 +146,18 @@ private void InitializeComponent()
             this.chkShowRelativeDate.Text = "Show relative date instead of full date";
             this.chkShowRelativeDate.UseVisualStyleBackColor = true;
             // 
+            // chkShowRepoCurrentBranch
+            // 
+            this.chkShowRepoCurrentBranch.AutoSize = true;
+            this.tlpnlGeneral.SetColumnSpan(this.chkShowRepoCurrentBranch, 2);
+            this.chkShowRepoCurrentBranch.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.chkShowRepoCurrentBranch.Location = new System.Drawing.Point(3, 26);
+            this.chkShowRepoCurrentBranch.Name = "chkShowRepoCurrentBranch";
+            this.chkShowRepoCurrentBranch.Size = new System.Drawing.Size(314, 17);
+            this.chkShowRepoCurrentBranch.TabIndex = 5;
+            this.chkShowRepoCurrentBranch.Text = "Show current branch of repositories";
```

Comment:
```suggestion
            this.chkShowRepoCurrentBranch.Text = "Shows current branch names in the dashboard and the recent repositories dropdown menu";
```

Reply:
Tooltip is obsolote then I guess since it says the same thing?

Also I would suggest just using 'Show' instead of 'Shows' here to make it consistent with 'Show current branch in Visual Studio'